etna/packages/etna-js/components/metis_exploration.tsx

```diff
@@ -45,12 +45,19 @@ function simplifyFolderListReturn(fullList: any) {
   }
 }
 
-export function PickBucket({ project_name=CONFIG.project_name, setBucket, bucket, label="Bucket", className}: {
+const contentUse = (folderContents: folderSummary, allowFiles: boolean) => {
+  return allowFiles ?
+    folderContents.folders.concat(folderContents.files) :
+    folderContents.folders
+}
+
+export function PickBucket({ project_name=CONFIG.project_name, setBucket, bucket, label="Bucket", className, disablePortal=true}: {
   project_name?: string;
   setBucket: any;
   bucket: string;
   label?: string;
   className?: string;
+  disablePortal?: boolean;
```

Comment:
This will always be defined now that there's a default?
```suggestion
  disablePortal: boolean;
```